### PR TITLE
Hipchat

### PR DESCRIPTION
**Pull request by jesseolsen**
Add HipChat support

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +36,7 @@
     "gulp-task-listing": "^1.0.1",
     "gulp-util": "^3.0.7",
     "hubot": "2.x",
+    "hubot-hipchat": "^2.12.0",
```
Comment (danielreed) [inline package.json]: Does this belong here Jesse? Not sure we need this.

File: README.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ the way you expect to work, with a single integrated view of your IT infrastruct
```
Comment (danielreed) [inline README.md]: Why is this file in the patch?

File: src/listener/server-hardware.js [hunk 3]
```diff
@@ -22,11 +22,14 @@ THE SOFTWARE.
```
Comment (danielreed) [inline src/listener/server-hardware.js]: Should this file be in the patch?

**Comment by jesseolsen** [comment]
Going to put up a cleaner PR...

**Pull request closed without merging**